fastchat/serve/monitor/copilot_arena.py

```diff
@@ -0,0 +1,86 @@
+import gradio as gr
+import pandas as pd
+import requests
+
+from fastchat.serve.monitor.monitor import recompute_final_ranking
+
+copilot_arena_leaderboard_url = "https://leaderboard-server.fly.dev/elo"
```

Comment:
should we keep this URL private? say load this from an env variable?

Reply:
Good idea, updated!